docs/pages/routes.md

```diff
@@ -133,6 +132,42 @@ This is equivalent to calling `app.add_page` with the same arguments.
 # Index is a special exception where it is available at both `/` and `/index`. All other pages are only available at their specified route.
 ```
 
+
+## Navigating Between Pages
+
+### Links
+
+Links are accessible elements used primarily for navigation. Use the `href` prop to specify the location for the link to navigate to.
+
+```python demo
+rx.link("Reflex Home Page.", href="https://reflex.dev")
+```
+
+You can also provide local links to other pages in your project without writing the full url.
+
+```python demo
+rx.link("Example", href="/docs/library",)
+```
+Check out the docs [here]({library.typography.link.path}) to learn more.
+
+### Redirect
+
+Redirect the user to a new path within the application using `rx.redirect()`.
+
+- `path`: The destination path or URL to which the user should be redirected.
+- `external`: If set to True, the redirection will open in a new tab. Defaults to `False`.
+
+```python demo
+rx.vstack(
```

Comment:
The main benefit of redirecting vs a regular link if that you can do it from the state-adding logic behind it. We should show off the difference here so the user has a better idea of what redirect can do,

Reply:
added